docs/langref/relay_pattern.rst

```diff
@@ -139,3 +237,125 @@ Domination
 **********
 
 Match child pattern, find a match for the parent pattern, insuring that the child ultimately dominates the parrent (i.e., no nodes outside the pattern use outputs of the parent), and that ever node betwen the child and the pattern matches the path pattern.
+
+Applications
+============
+
+The pattern language provides not only the pattern matching but also pattern processing. The later
```

Comment:
"The Pattern Language provides pattern processing in addition to matching capabilities. Here we introduce two
pattern processing approaches and provide some examples."

I don't think the middle sentence is necessary because you go into much greater detail below on the two approaches?

Reply:
Make sence. Removed.